geometry_msgs/Pose[2] poses
float32 g140
float32 g3f_1
float32 g3f_2
float32 g3f_m
float32 g3f_p